### PR TITLE
Feat/onboard native usdc

### PR DESCRIPTION
**Pull request by Rozengarden**
listing doesn't work, telling that reserve is already initialised

### CODE REVIEW

File: src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Polygon_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('polygon'), 50786431);
+    proposal = new AaveV3Polygon_OnboardNativeUSDCToAaveV3Markets_20231205();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_defaultProposalExecution() public {
+    startHoax(USDC_WHALE);
```
Comment (sakulstra) [inline src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Polygon_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol:29]: hoaxing here is fine and i guess we can even add to the template

File: src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Polygon_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    defaultTest(
+      'AaveV3Polygon_OnboardNativeUSDCToAaveV3Markets_20231205',
+      AaveV3Polygon.POOL,
+      address(proposal)
+    );
+  }
+
+  function test_collectorHasnUSDCFunds() public {
+    startHoax(USDC_WHALE);
+    IERC20(proposal.USDCn()).transfer(GovernanceV3Polygon.EXECUTOR_LVL_1, 10 ** 6);
```
Comment (sakulstra) [inline src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Polygon_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol:40]:
```suggestion
```

please don't hoax here, this remove the sole purpose of this test - as long as it is failing it cannot be merged.

File: src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/OnboardNativeUSDCToAaveV3Markets.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+- Tests: [AaveV3Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Polygon_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol), [AaveV3Optimism](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Optimism_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol), [AaveV3Arbitrum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Arbitrum_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol), [AaveV3Base](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Base_OnboardNativeUSDCToAaveV3Markets_20231205.t.sol)
+- [Snapshot](No snapshot for Direct-to-AIP)
```
Comment (brotherlymite) [inline src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/OnboardNativeUSDCToAaveV3Markets.md:175]:
```suggestion
- Snapshot: No Snapshot for Direct-to-AIP
```

File: src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/OnboardNativeUSDCToAaveV3Markets.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+## Specification
+
+### Assets listing
+
+The table below illustrates the configured risk parameters for **USDC** on Polygon POS
+
+| Parameter                          |                                      Value |
+| ---------------------------------- | -----------------------------------------: |
+| Isolation Mode                     |                                       true |
```
Comment (brotherlymite) [inline src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/OnboardNativeUSDCToAaveV3Markets.md:41]:
```suggestion
| Isolation Mode                     |                                       false |
```

File: src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/OnboardNativeUSDCToAaveV3Markets.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+| Flashloanable                      |                                    ENABLED |
+| Siloed Borrowing                   |                                   DISABLED |
+| Borrowable in Isolation            |                                   DISABLED |
+| Oracle                             | 0xfE4A8cc5b5B2366C1B58Bea3858e81843581b2F7 |
+
+The table below illustrates the configured risk parameters for **USDC** on base
+
+| Parameter                          |                                      Value |
+| ---------------------------------- | -----------------------------------------: |
+| Isolation Mode                     |                                       true |
```
Comment (brotherlymite) [inline src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/OnboardNativeUSDCToAaveV3Markets.md:71]:
```suggestion
| Isolation Mode                     |                                       false |
```

File: src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Arbitrum_OnboardNativeUSDCToAaveV3Markets_20231205.sol [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    returns (IAaveV3ConfigEngine.CollateralUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.CollateralUpdate[]
+      memory collateralUpdate = new IAaveV3ConfigEngine.CollateralUpdate[](2);
+
+    collateralUpdate[0] = IAaveV3ConfigEngine.CollateralUpdate({
+      asset: AaveV3ArbitrumAssets.USDC_UNDERLYING,
+      ltv: 77_00,
+      liqThreshold: 80_00,
+      liqBonus: 5_00,
```
Comment (brotherlymite) [inline src/20231205_Multi_OnboardNativeUSDCToAaveV3Markets/AaveV3Arbitrum_OnboardNativeUSDCToAaveV3Markets_20231205.sol:72]: I think the current liquidation bonus already is 5% so we can use `KEEP_CURRENT` here

**Review by sakulstra — APPROVED** [review]

**Pull request merged**